account_fiscal_year/__manifest__.py

```diff
@@ -16,6 +16,7 @@
     'data': [
         'data/date_range_type.xml',
         'views/date_range_type.xml',
+        'security/ir.model.access.csv',
```

Comment:
Shouldn't it be at top of data?

Reply:
I don't see any reason why the order should be important in this particular module.